src/server/luaentity_sao.cpp

```diff
@@ -294,11 +309,13 @@ void LuaEntitySAO::getStaticData(std::string *result) const
 	writeF1000(os, m_rotation.Y);
 
 	// version2. Increase this variable for new values
-	writeU8(os, 1); // PROTOCOL_VERSION >= 37
+	writeU8(os, 2); // PROTOCOL_VERSION >= 37
```

Comment:
i think the reference to protocol ver here doesn't make much sense, it's map data.
```suggestion
	writeU8(os, 2);
```